spec/form_spec.cr

```diff
@@ -203,6 +203,14 @@ describe "LuckyRecord::Form" do
       form.name.value.should eq "Joe"
       form.name.errors.should eq ["wrong"]
     end
+
+    it "Field#value returns nil on empty strings" do
+      empty_string = LuckyRecord::Field.new(:blank, nil, " ", "test_form")
+      empty_string.value.should be_nil
+
+      empty_array = LuckyRecord::Field.new(:empty_array, nil, [] of String, "test_form")
```

Comment:
Could you use names arguments so it’s a bit easier to see what they each option is? E.g. Field.new(name: :empty_array, ...)